projects/go-lib/src/lib/components/go-select/go-select.component.ts

```diff
@@ -56,6 +56,10 @@ export class GoSelectComponent implements OnInit {
     this.control.patchValue(this.items.map((item: any) => this.bindValue ? item[this.bindValue] : item));
   }
 
+  onRemoveAll(): void {
+    this.control.reset();
+  }
```

Comment:
Should we write a test for this method?

Reply:
@grahamhency test case done. Please check